<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPasswordDialog</class>
 <widget class="QDialog" name="AddPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/edit password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="keyEdit">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password/value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwEdit1">
     <property name="maxLength">
      <number>512</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Repeat password/value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwEdit2">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>512</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showHideButton">
       <property name="text">
        <string>Show/hide</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generatePasswordButton">
       <property name="text">
        <string>Generate password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Comments</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="commentsEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPasswordDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPasswordDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
